plugins/parsers/grok/parser_test.go

```diff
@@ -915,17 +915,17 @@ func TestSyslogTimestamp(t *testing.T) {
 		{
 			name:     "two digit day of month",
 			line:     "Sep 25 09:01:55 value=42",
-			expected: time.Date(2018, time.September, 25, 9, 1, 55, 0, time.UTC),
+			expected: time.Date(2019, time.September, 25, 9, 1, 55, 0, time.UTC),
```

Comment:
Let's use `time.Now().Year()` so they don't fail in 2020.

Reply:
Fixed